relay-aws-extension/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+//! AWS extension implementation for Sentry's AWS Lambda layer.
```

Comment:
Since these docs will be deployed to https://getsentry.github.io/relay/ and serve as reference material going forward, could you add a bit more description around what this crate is about, why it exists, and what it does high-level? You can also link to the actor struct for more specific information.

See an example of this here: https://getsentry.github.io/relay/relay_metrics/